modules/javafx.web/src/main/native/Source/WebCore/page/DOMWindow.cpp

```diff
@@ -845,22 +845,18 @@ ExceptionOr<Storage*> DOMWindow::localStorage()
         return Exception { SecurityError };
 
     auto* page = document->page();
-    // FIXME: We should consider supporting access/modification to local storage
-    // after calling window.close(). See <https://bugs.webkit.org/show_bug.cgi?id=135330>.
-    if (!page || !page->isClosing()) {
-        if (m_localStorage)
-            return m_localStorage.get();
-    }
-
```

Comment:
This will change the behavior for the case where page is null or where the page is valid, but not closing. I think you should partially revert this part of the fix, restoring it as follows:

```
    if (m_localStorage)
        return m_localStorage.get();
```